proposals/all_proposals.md

```diff
@@ -0,0 +1,658 @@
+
+3. Which models of data ownership and governance can democratise the production and usage of data? When should data be open?
+
+Personal data needs to be protected with the highest protections we can achieve (Convention 108, GDPR….). Non-personal data should be open whenever possible so that everyone can re-use it. Specially, data produced by public institutions needs to be made open by default. We do not recommend models based on “data ownership” that envision the possibility to to trade personal data for services and benefits. It would affect the vunlnerable populations that have less resources and digital literacy and therefore enable forced trade of fundamental rights.
+
+4. Which regulations can constrain the growing Internet monopolies (Google, Amazon, …) and make them accountable to democratic decisions? (without creating ridiculous constrains for the small scale and non-profit internet ecosphere)
+
+Convention 108 and GDPR for data protection; ePrivacy Regulation for privacy; modernisation of copyright laws and in the medium-term a revision of the Berne Convention for copyright towards ensuring that the noncommercial user activity that does not infringe interestes of creators is not criminalised.
+
+5. How can alternative economic models such as co-ops and commons projects be fostered in all technology?
+
+Economic models such as co-ops and commons projects face great opportunities of growth online because they are by nature decentralised and participatory. What endangeres their thriving is the increasingly centralised model of services that is labeled as sharing aconomy but in practice is a rental model where rights and property are privatised (by infrastructure providers) while the risk and costs are socialised (by actual service providers and their clients to an extent). The so called “uberisation” of participatory economy should be resisted by building services that are truly owned by their providers who also collectively own the technological infrastructure though which users access these services. In that way the infrastructure-and-service provider is participating in the benefits, including ownership of the shares and the risks are managed collectively without an incentive to transfer them to frontline service providers. Open ecosystems, be it for content (free and open licensing) or for providing infrastructure (free and open software) ensure that the concentration is prevented by their non-exclusive and non-proprietary character.
```

Comment:
possible connection to labor intelligence / coop-supporting institute proposal - kate